hack/local-up-cluster.sh

```diff
@@ -285,6 +285,10 @@ function start_apiserver {
     if [[ -n "${ALLOW_ANY_TOKEN:-}" ]]; then
       anytoken_arg="--insecure-allow-any-token "
     fi
+    authorizer_arg=""
+    if [[ -n "${ENABLE_RBAC:-}" ]]; then
```

Comment:
> I would have expected that if ENABLE_RBAC is "false" then we wouldn't use this.

The other condition checks in the same method don't make the distinction.  Do you feel strongly?